libs/block-editor/src/lib/extensions/dragHandler.extension.ts

```diff
@@ -95,14 +96,17 @@ export const DragHandler = (viewContainerRef: ViewContainerRef) => {
                 dragHandler.setAttribute('draggable', 'true');
                 dragHandler.addEventListener('dragstart', (e) => dragStart(e, editorView));
                 dragHandler.style.display = 'none';
-                document.body.appendChild(dragHandler);
+                editorView.dom.parentElement.appendChild(dragHandler);
             }
 
             return [
                 new Plugin({
                     key: new PluginKey('dragHandler'),
                     view: (editorView) => {
-                        bindEventsToDragHandler(editorView);
+                        setTimeout(() => {
+                            bindEventsToDragHandler(editorView);
+                        }, 0);
```

Comment:
Why?

Reply:
Maybe: https://stackoverflow.com/questions/38709923/why-is-requestanimationframe-better-than-setinterval-or-settimeout

Reply:
Done.